runelite-client/src/main/java/net/runelite/client/plugins/microbot/tutorialisland/TutorialIslandScript.java

```diff
@@ -332,20 +342,29 @@ public void CombatGuide() {
             if (isInDialogue()) return;
             if (Microbot.getVarbitPlayerValue(281) == 500) {
                 Rs2Walker.walkTo(new WorldPoint(3111, 9526, Rs2Player.getWorldLocation().getPlane()));
-                Rs2GameObject.interact("Ladder","Climb-up");
+                sleep(5000);
+                sleepUntil(() -> !Rs2Player.isMoving());
+                System.out.println("We're done moving...");
```

Comment:
Moving to ladder, as had problems with finding the ladder unless it was close for some reason.